templates/page-styleguide.php

```diff
@@ -19,7 +19,7 @@
 	<div class="uikit__content">
 
 		<?php
-			$colors = get_colors( '/assets/css/frontend/global/variables.css' );
+			$colors = \TenUpScaffold\Template_Tags\get_colors( '/assets/css/frontend/global/variables.css' );
```

Comment:
@christianc1 does 10up recommend using function aliasing? 

```
use function My\Full\functionName as func;

func(); // calls function My\Full\functionName
```

I assume by your note above that since you did not include `as get_colors;` and `as adjust_brightness()` that it is unlikely advisable. I'm wondering what the logic is and if I should open an Issue to update the Engineering Best Practices to clarify why?

Reply:
I guess if another Engineer came along and tried to copy/paste an unknowingly aliased function to another file then it might cause some confusion (and subsequent unnecessary debugging). 

Therefore, that is why it is likely ill-advised to use aliasing? Anything else?

Reply:
@colorful-tones ,

We don't need function aliasing in this scenario.  Here's the use case for using an alias:

Say we have a plugin called Featured Post that lets you set a post to feature, and has defined the following function in a namespace:
`TenupFeaturedPost\get_the_title();`

We can't have the declaration ...
```php
use function TenupFeaturedPost\get_the_title;
````
... in the same file where we also use the function `get_the_title()` from WordPress core.  In this instance, we'd use aliasing:
```php
use function TenupFeaturedPost\get_the_title as get_the_featured_title;
```
That way, we could use both functions in the same file.

Reply:
Yeah, I guess the most obvious use case for aliasing for me was saving some keystrokes, e.g. `\TenUpScaffold\Utility\get_colors()` vs `get_colors()`, but I can see how efficiency does not necessarily outweigh readability and reusability in that case.

I had not thought about use case that you described above (`get_the_title()`) and glad you clarified. Totally make sense.

Thanks @christianc1

Reply:
Thanks @colorful-tones, glad I was able to clarify their use case.